lib/components/admin/field-trip-details.js

```diff
@@ -0,0 +1,215 @@
+  InlineHeader,
+  P,
+  Text,
+  Val
+} from './styled'
+import TripStatus from './trip-status'
+import Updatable from './updatable'
+import {
+  getGroupSize,
+  GROUP_FIELDS,
+  PAYMENT_FIELDS,
+  TICKET_TYPES
+} from '../../util/call-taker'
+
+/**
+ * Shows the details for the active Field Trip Request.
+ */
+class FieldTripDetails extends Component {
+  _editSubmitterNotes = (val) => this.props.editSubmitterNotes(this.props.request, val)
+
+  _getRequestLink = (path, isPublic = false) =>
+    `${this.props.datastoreUrl}/${isPublic ? 'public/' : ''}fieldtrip/${path}?requestId=${this.props.request.id}`
+
+  _onCloseActiveFieldTrip = () => this.props.setActiveFieldTrip(null)
+
+  _onClickCancel = () => {
+    const {request, setRequestStatus} = this.props
+    if (confirm('Are you sure you want to cancel this request? Any associated trips will be deleted.')) {
+      setRequestStatus(request, 'cancelled')
+    }
+  }
+
+  render () {
+    const {
+      addFieldTripNote,
+      callTaker,
+      dateFormat,
+      deleteFieldTripNote,
+      request,
+      setRequestGroupSize,
+      setRequestPaymentInfo
+    } = this.props
+    if (!request) return null
+    const {
+      id,
+      invoiceRequired,
+      notes,
+      schoolName,
+      submitterNotes,
+      teacherName,
+      ticketType,
+      travelDate
+    } = request
+    const {fieldTrip} = callTaker
+    const defaultPosition = {...fieldTrip.position}
+    const internalNotes = []
+    const operationalNotes = []
+    notes && notes.forEach(note => {
+      if (note.type === 'internal') internalNotes.push(note)
+      else operationalNotes.push(note)
+    })
+    defaultPosition.x = defaultPosition.x - 460
+    defaultPosition.y = defaultPosition.y - 100
+    const travelDateAsMoment = moment(travelDate)
+    const total = getGroupSize(request)
+    return (
+      <DraggableWindow
+        draggableProps={{ defaultPosition }}
+        footer={
+          <div style={{padding: '5px 10px 0px 10px'}}>
+            <DropdownButton
+              aria-label='Field Trip Menu'
+              bsSize='xsmall'
+              dropup
+              id='field-trip-menu'
+              title='View'
+            >
+              <MenuItem
+                href={this._getRequestLink('feedbackForm', true)}
+                target='_blank'
+              >
+                <Icon type='link' /> Feedback link
+              </MenuItem>
+              <MenuItem
+                href={this._getRequestLink('receipt')}
+                target='_blank'
+              >
+                <Icon type='file-text-o' /> Receipt link
+              </MenuItem>
+            </DropdownButton>
+            <Button
+              bsSize='xsmall'
+              bsStyle='danger'
+              className='pull-right'
+              onClick={this._onClickCancel}
+            >
+              Cancel request
+            </Button>
+          </div>
+        }
+        header={
+          <h4 style={{marginBottom: 0}}>
```

Comment:
Would it make better sense if this was a (styled) `<h3>`? It would fit, considering that you are using `<h4>` for the different sections under this window, and `<h5>`for the instructor and operator notes.